autoscheduler/scheduler/tests/api_tests.py

```diff
@@ -112,9 +112,11 @@ def test_route_scheduling_generate_is_correct(self, create_schedules_mock):
             "includeFull": True,
         }
 
-        expected = [
-            [SectionSerializer(section).data for section in self.sections]
-        ]
+        expected = {
+            'schedules': [[SectionSerializer(section).data for section in self.sections]],
+            'message': '',

```

Comment:
It doesn't really matter either way imo, I wanted it to always be a string since empty string and null work the same way in javascript and python. Can change if you care about it though